src/aspire/source/simulation.py

```diff
@@ -42,10 +50,31 @@ def __init__(
         noise_filter=None,
     ):
         """
-        A Cryo-EM simulation
-        Other than the base class attributes, it has:
-
-        :param angles: A n-by-3 array of rotation angles
+        A `Simulation` object that supplies images along with other parameters for image manipulation.
+
+        :param L: Resolution of projection images (integer).
+            Default is 8 or vols.resolution if a volume is provided.
+        :param n: The number of images to generate (integer).
+        :param vols: A Volume object. Default is a LegacyVolume.
+        :param states: A 1d array of n integers in the interval [0, C). The i'th integer indicates
+            the volume used to produce the i'th projection image. Default is a random set.
+        :param unique_filters: A list of Filter objects to be applied to projection images.
+        :param filter_indices: A 1d array of n integers indicating the unique_filter indices associated
+            with each image. Default is a random set of filter indices.
+        :param offsets: A n-by-2 array of coordinates to offset the images. Default is a normally
```

Comment:
If the option exists, mention you can set to zero to get no offsets. I'm not sure if this is a feature introduced by #768 but either way we'll want to add it here

Reply:
It's not currently a feature, so I'll leave it out for now.